.github/workflows/settings-2-deploy.yml

```diff
@@ -45,7 +45,7 @@ jobs:
           updates=$(jq --compact-output --raw-output \
             --arg env "${{ inputs.deployment-environment }}" \
             --arg type "${{ inputs.spack-type }}" \
-            '.deployment[$env][$type] | to_entries[] | "\(.key) \(.value.spack-config)"' \
+            '.deployment[$env][$type] | to_entries[] | "\(.key) \(.value."spack-config")"' \
```

Comment:
Does double nesting of `"` quotes work?

Reply:
Yes, verified locally:
```bash
$ jq -cr --arg env "Gadi" --arg type "Prerelease" '.deployment[$env][$type] | to_entries[] | "\(.key) \(.value."spack-config")"' config/settings.json 
0.22 2024.07.05
```

Reply:
Wow. All other languages I've ever used would not like this at all. `jq` you crazy outlier. Never let anyone change you.